components/GymPlan/GymPlan.js

```diff
@@ -0,0 +1,19 @@
+import WorkoutSet from './WorkoutSet';
+
+export default function GymPlan() {
+   function handleSubmit(event) {
+      event.preventDefault();
```

Comment:
Remember to use the handle submit function to access the values of your input fields, so you can render them :)